### ISSUE TITLE
No indications on the phoneweb 

### ISSUE THREAD

**Issue by vhatzist** [gmaruzz/saraphone#30]

**Comment by vhatzist** [comment]
Hello, 

Saraphone performs a successful subscription and receives a NOTIFY packet like the one below, but there is no indication on the web page that the monitored phone is on or off hook.

The NOTIFY is: 

NOTIFY sip:022mlhi3@eqsdpm621emt.invalid;transport=wss SIP/2.0
Max-Forwards: 10
Record-Route: <sip:<server IP>:8443;transport=ws;r2=on;lr=on>
Record-Route: <sip:<server IP>;r2=on;lr=on>
Via: SIP/2.0/WSS <server IP>:8443;branch=z9hG4bK16a1.adec128d8d18e4bcfdaf6f201fc2cfa1.0
Via: SIP/2.0/UDP 127.0.0.1;rport=52518;received=<server IP>;branch=z9hG4bKRyBvGeAcAZ
From: <sip:1127976@my_domain>;tag=159880548411955
To: <sip:1127975@my_domain>;tag=1gcpdooeis
Contact: <sip:1127976@my_domain>
Call-ID: 1b71hh376eh4doekv5os
CSeq: 4 NOTIFY
User-Agent: Presence server
Event: presence
Subscription-State: active;expires=114
Content-Type: application/pidf+xml
Content-Length: 521
X-Enswitch-RURI: sip:1127975@my_domain;gr=urn:uuid:db1e31b0-b5b3-4481-9a6f-b216c46ffd9b;alias=<browser IP>~59459~6
X-Enswitch-Source: <server IP>:52518;transport=udp

<?xml version="1.0" encoding="ISO-8859-1"?><presence xmlns="urn:ietf:params:xml:ns:pidf" xmlns:pp="urn:ietf:params:xml:ns:pidf:person" xmlns:es="urn:ietf:params:xml:ns:pidf:rpid:status:rpid-status" xmlns:ep="urn:ietf:params:xml:ns:pidf:rpid:rpid-person" entity="sip:1127976@my_domain"><pp:person><status></status></pp:person><note>Ready</note><tuple id="1127975"><contact priority="1">sip:1127976@my_domain;tag=159880548411955</contact><status><basic>open</basic></status></tuple></presence>

Is there a problem with the format of the message or the BLF labels and buttons should be named otherwise for this to work? Or perhaps additional Javascript coding is needed for this?

In saraphone.html I have used:

<button class="btn btn-sm btn-primary btn-default" data-inline="true" id="pres1btn" style="margin-bottom: 5px; width: 150px !important;" onclick="audio1.play ( )"><span id="ispresent1">1127976</span></button>

as a BLF button to monitor the state of phone 1127976.

The web phone is registering as phone 1127975.

Thank you in advance for any assistance.

**Comment by gmaruzz** [comment]
hello,

if the phone is on-hook the BLF button stay green

when the monitored phone is in a call, the button changes?

on webphone, if you click on ther BLF button, it calls the monitored phone?

can you attach a complete sip trace from webphone start (reload), so there is registration, subscriptions, notifys, etc etc then do a call from monitored, so there will be trace of the further notifys ?

thanks!

**Comment by vhatzist** [comment]
Hi @gmaruzz ,

Sorry for the late reply. To answer your questions:

- The BLF button does not change color, it always remains the same grey color (image BLF_button.jpg) irrespective of whether it is on-hook or off-hook. 

![BLF_button](https://user-images.githubusercontent.com/14276813/94467707-fed08780-01cb-11eb-9e9c-3df65c9d361a.jpg)

- There is no button change when the monitored phone is in a call.

- On the webphone, if we click on the BLF button it calls the monitored phone without problems. 

I have attached 2 files, the first contains the registration and subscriptions with an initial NOTIFY about the monitored phone's status. The second contains NOTIFY messages while the monitored phone was calling another phone. The hostname and IP address have been changed for security reasons.

[register_and_subscribe.txt](https://github.com/gmaruzz/saraphone/files/5294115/register_and_subscribe.txt)
[notify.txt](https://github.com/gmaruzz/saraphone/files/5294127/notify.txt)
